modules/caddyhttp/rewrite/caddyfile.go

```diff
@@ -49,89 +46,63 @@ func parseCaddyfileRewrite(h httpcaddyfile.Helper) (caddyhttp.MiddlewareHandler,
 	return rewr, nil
 }
 
-// parseCaddyfileStripPrefix sets up a handler from Caddyfile tokens. Syntax:
+// parseCaddyfileURI sets up a handler for manipulating (but not "rewriting") the
+// URI from Caddyfile tokens Syntax:
```

Comment:
I know it's probably the pettiest thing ever to "code" review, but should there be a full stop before the word Syntax here? :D

Reply:
Ruthless!

Reply:
Haha, I'm four walls mate!

Reply:
Is that aussie for "crazy"?

Reply:
It's English for "****ing roofless!"